evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -76,6 +78,116 @@ fn get_code(path: &str) -> Result<Bytes, Bytes> {
     }
 }
 
+// struct BytesResult(Result<Bytes, Bytes>);
+
+// impl<T: AbiEncode, E: ToString> From<Result<T, E>> for BytesResult {
+//     fn from(src: Result<T, E>) -> Self {
+//         BytesResult(src.map(|v| v.encode().into()).map_err(|e| e.to_string().encode().into()))
+//     }
+// }
+
+fn set_env(key: &str, val: &str) -> Result<Bytes, Bytes> {
+    // `std::env::set_var` may panic in the following situations
+    // ref: https://doc.rust-lang.org/std/env/fn.set_var.html
+    if key.is_empty() {
+        Err("Environment variable key can't be empty".to_string().encode().into())
+    } else if key.contains('=') {
+        Err("Environment variable key can't contain equal sign `=`".to_string().encode().into())
+    } else if key.contains('\0') {
+        Err("Environment variable key can't contain NUL character `\0`".to_string().encode().into())
```

Comment:
```suggestion
        Err("Environment variable key can't contain NULL character `\0`".to_string().encode().into())
```

Reply:
Was following `std::env::set_var` doc which uses the term *NUL character*. [ref](https://doc.rust-lang.org/std/env/fn.set_var.html)

In Rust reference on Tokens, it also states that "The *null escape* is the character `U+0030` (`0`) and denotes the Unicode value `U+0000` (*NUL*)". [ref](https://doc.rust-lang.org/reference/tokens.html)